dockstore-common/src/main/java/io/dockstore/common/yaml/DockstoreYamlHelper.java

```diff
@@ -21,6 +22,7 @@
 import org.yaml.snakeyaml.constructor.Constructor;
 import org.yaml.snakeyaml.introspector.Property;
 import org.yaml.snakeyaml.introspector.PropertyUtils;
+import scala.tools.nsc.transform.patmat.Logic;
```

Comment:
additional imports seem a bit superfluous if nothing else is changing in this file